spec/index.bs

```diff
@@ -13714,6 +13726,57 @@ In WebGPU, it does so by [$Replace the drawing buffer|replacing the drawing buff
     validation will fail inside {{GPUCanvasContext/getCurrentTexture()}}.
 </div>
 
+<h3 id=gpucanvastonemappingmode data-dfn-type=enum>`GPUCanvasToneMappingMode`
+</h3>
+
+This enum specifies how color values are displayed to the screen.
+
+<dl dfn-type=enum-value dfn-for=GPUCanvasToneMappingMode>
+    : <dfn>"standard"</dfn>
+    ::
+        Color values within the standard dynamic range of the screen are unchanged and
+        all other color values are projected to the standard dynamic range of the screen.
+        This projection is often accomplished by clamping color values in the color space
+        of the screen to the `[0, 1]` interval.
+
+        <div class=example>
+            For example, suppose that the value `(1.035, -0.175, -0.140)` is written to an
+            `'srgb'` canvas.
+
+            If this is presented to an sRGB screen, then this will be converted to sRGB
+            (which is a no-op, because the canvas is sRGB), and then will be clamped to
+            the sRGB value `(1.0, 0.0, 0.0)`.
+
+            If this is presented to a Display P3 screen, then this will be converted to
+            the value `(0.948, 0.106, 0.01)` in the Display P3 color space, and no
+            clamping will be needed.
+        </div>
+
+    : <dfn>"extended"</dfn>
+    ::
+        Color values in the extended dynamic range of the screen are unchanged and all
+        other color values are projected to the extended dynamic range of the sceen. This
+        projection is often accomplished by clamping color values in the color space of
+        the screen to the interval of values that the screen is capable of displaying,
+        which may include values greater than `1`.
+
+        <div class=example>
+            For example, suppose that the value `(2.5, -0.15, -0.15)` is written to an
+            `'srgb'` canvas.
+
+            If this is presented to an sRGB screen that is capable of displaying values
+            in the `[0, 4]` interval in sRGB space, then this will be converted to sRGB
+            (which is a no-op, because the canvas is sRGB), and then will be clamped to
+            the sRGB value `(2.5, 0.0, 0.0)`.
```

Comment:
```suggestion
            (which is a no-op, because the canvas is sRGB), then projected into the display's space.
            Using component-wise clamping, this results in the sRGB value `(2.5, 0.0, 0.0)`.
```